awesome_dashboard/static/src/dashboard/dashboard.js

```diff
@@ -0,0 +1,64 @@
+/** @odoo-module **/
+
+import { Component, useState, useEffect } from "@odoo/owl";
+import { Layout } from "@web/search/layout";
+import { useService } from "@web/core/utils/hooks";
+import { AwesomeDashboardItem } from "@awesome_dashboard/dashboard/dashboard_item";
+import { registry } from "@web/core/registry";
+import { CogMenu } from "@web/search/cog_menu/cog_menu";
+import { AwesomeDashboardConfigDialog } from "@awesome_dashboard/dashboard/dialogs/dashboard_config_dailog";
```

Comment:
```suggestion
import { AwesomeDashboardConfigDialog } from "@awesome_dashboard/dashboard/dialogs/dashboard_config_dialog";
```
Misspelling, happens 🤷
Don't forget to rename your file too